integrationtests/git-server/modules/columnchart/columnchart.py

```diff
@@ -26,8 +26,8 @@ def python_format_to_d3_tick_format(python_format: str) -> str:
     # Formatter.parse() returns Iterable[(literal, field_name, format_spec,
     # conversion)]
     specifier = next(Formatter().parse(python_format))[2]
-    if not specifier or specifier[-1] not in 'bcdoxXneEfFgGn%':
-        specifier += 'r'
+    if not specifier or specifier[-1] not in "bcdoxXneEfFgGn%":
```

Comment:
Aaaah -- Black should not reformat these. This was a bug in `master` and I've just fixed it.

Please merge `master` and then undo all edits in `integrationtests/git-server/modules/`. Black won't complain about them again after the update I just pushed to master.